sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -327,7 +327,7 @@ export class MessageSender extends LinkEntity {
         if (this._sender!.sendable()) {
           try {
             this._sender!.sendTimeoutInSeconds =
-              (this._retryOptions.timeoutInMs - timeTakenByInit) / 1000;
+              (this._retryOptions.timeoutInMs! - timeTakenByInit) / 1000;
```

Comment:
In #8685, we added a guard to ensure this does not result in -ve value in Event Hubs. Can we include that in this PR for Service Bus

Reply:
Added